vite-project/src/scouter/querytypes/RadioQuery.tsx

```diff
@@ -1,28 +1,24 @@
 import React from "react";
+import { localStorageTabName } from "../ScouterQuery";
 
 interface RadioQueryProps {
   name: string;
   required: boolean | undefined;
-  initialValue: string;
   list: string[];
 }
 
-const RadioQuery: React.FC<RadioQueryProps> = ({
-  name,
-  required,
-  initialValue,
-  list,
-}) => {
+const RadioQuery: React.FC<RadioQueryProps> = ({ name, required, list }) => {
+  const localStorageKey = localStorageTabName + name;
   return list?.map((item, index) => (
     <React.Fragment key={index}>
-      {item === initialValue ? (
+      {item === localStorage.getItem(localStorageKey) ? (
         <input
           type="radio"
           id={item}
           name={name}
           value={item}
           required={required}
-          onChange={() => localStorage.setItem(name, item)}
+          onChange={() => localStorage.setItem(localStorageKey, item)}
           defaultChecked
         />
```

Comment:
having such large item inside ternary operator is hard to read, consider moving to separate variable